core/src/test/java/org/infinispan/globalstate/AbstractGlobalStateRestartTest.java

```diff
@@ -89,6 +87,9 @@ void createStatefulCacheManager(String id, boolean clear) {
 
    protected void applyCacheManagerClusteringConfiguration(String id, ConfigurationBuilder config) {
       applyCacheManagerClusteringConfiguration(config);
+
+      config.persistence().addSoftIndexFileStore();
```

Comment:
This is moved so the `ThreeNodeGlobalStatePartialRestartTest#applyCacheManagerClusteringConfiguration` takes precedence.